content/en/docs/tasks/observability/distributed-tracing/opentelemetry/index.md

```diff
@@ -0,0 +1,181 @@
+
+{{< text bash >}}
+$ kubectl create namespace otel-collector
+{{< /text >}}
+
+1.  Deploy the OpenTelemetry Collector. You can use this example configuration as an starting point:
+[`otel.yaml`]({{< github_blob >}}/samples/open-telemetry/otel.yaml)
+
+{{< text bash >}}
+$ kubectl -n otel-collector apply -f otel.yaml
+{{< /text >}}
+
+## Installation
+
+All tracing options can be configured globally via `MeshConfig`.
+To simplify configuration, it is recommended to create a single YAML file
+which you can pass to the `istioctl install -f` command.
+
+### Exporting via gRPC
+
+In this example, traces will be exported via OTLP/gRPC to the OpenTelemetry Collector.
+
+{{< text bash >}}
+$ cat <<'EOF' > tracing.yaml
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+spec:
+  meshConfig:
+    extensionProviders:
+    - name: otel-tracing
+      opentelemetry:
+        port: 4317
+        service: opentelemetry-collector.otel-collector.svc.cluster.local
+EOF
+$ istioctl install -f ./tracing.yaml --skip-confirmation
+kubectl label namespace default istio-injection=enabled
+{{< /text >}}
+
+### Exporting via HTTP
+
+In this example, traces will be exported via OTLP/HTTP to the OpenTelemetry Collector.
+
+{{< text bash >}}
+$ cat <<'EOF' > tracing.yaml
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+spec:
+  meshConfig:
+    extensionProviders:
+    - name: otel-tracing
+      opentelemetry:
+        port: 4318
+        service: opentelemetry-collector.otel-collector.svc.cluster.local
+        http:
+          path: "v1/traces"
+          timeout: 10s
+          headers:
+          - name: "my-custom-header"
+            value: "some-value"
+EOF
+$ istioctl install -f ./tracing.yaml --skip-confirmation
+kubectl label namespace default istio-injection=enabled
+{{< /text >}}
+
+{{< tip >}}
+You can also send traces directly to your tracing back-end of choice via HTTP, without needing a Collector.
+For that, you will need to first define a [ServiceEntry](/docs/reference/config/networking/service-entry/).
+{{< /tip >}}
+
+## Enable tracing for mesh via Telemetry API
+
+Enable tracing by applying the following configuration:
+
+{{< text bash >}}
+$ kubectl apply -f - <<EOF
+apiVersion: telemetry.istio.io/v1alpha1
+kind: Telemetry
+metadata:
+  name: otel-demo
+  namespace: istio-system
+spec:
+  tracing:
+  - providers:
+    - name: otel-tracing
+    randomSamplingPercentage: 100
+    customTags:
+      "my-attribute":
+        literal:
+          value: "default-value"
+EOF
+{{< /text >}}
+
+## Deploy the Bookinfo Application
+
+Deploy the [Bookinfo](/docs/examples/bookinfo/#deploying-the-application) sample application.
+
+## Generating traces using the Bookinfo sample
+
+1.  When the Bookinfo application is up and running, access `http://$GATEWAY_URL/productpage`
+one or more times to generate trace information.
+
+1.  You can look at the Collector logs to verify traces are arriving.
+The Collector logs will contain something like:
```

Comment:
IIRC the log is shown with the debug log level setting. Do we need to mention this in case someone is not seeing the log?

Reply:
Good point. I think it is still helpful because the collector config used in the example (same as we have in our samples) export the traces to the console and to Jaeger, but we can't be sure if Jaeger is deployed. So the console is the most easy way to check. 

I will add a better explanation that the console exporter must be configured in the collector to be able to see the traces.